### DISCUSSION TITLE
Difference between F-Droid APKs

### DISCUSSION THREAD

**Discussion by culjonito** [stratumauth/app#1233] [Q&A] [answered]
Hi folks

Can someone pls explain the difference between the following APKs?

com.stratumauth.app.apk
com.stratumauth.app.fdroid.apk
com.stratumauth.app-arm64-v8a.fdroid.apk

![xuQDoYbNnm](https://github.com/user-attachments/assets/42c74d04-b542-423c-aa57-a5ed6992a2f8)

Thank you!

**Comment by jamie-mh** [comment, marked as answer]
Hi,

A comparison of some releases is shown on the [website](https://stratumauth.com/download).

- **com.stratumauth.app**: the Google Play release with Wear OS support (all device types)
- **com.stratumauth.app.fdroid**: the F-Droid release **without** Wear OS support (all device types)
- **com.stratumauth.app-arm64-v8a.fdroid**: the F-Droid release but only for ARM64 (most devices)
